### ISSUE TITLE
Rename `IsHPC` to `UseHPC` in `DoEchelonTransformationMatBlockwise`

### ISSUE THREAD

**Issue by ssiccha** [lbfm-rwth/GaussPar#159]

**Comment by emmakatherina** [comment]
I don't really understand what you mean.
Changing `IsHPC` to `UseHPC` sounds good to me. Do we want to change it in `Chief`, too?

**Comment by ssiccha** [comment]
Yeah, I didn't write my thoughts up clearly. What I meant is: We use, what is now called `IsHPC`, to tell the algorithm whether to use the parallel or the sequential code path. I suggested to rename that variable to `UseHPC`.

Now that I think about it though, I think we should postpone this issue until we're done with milestone 0.2. It may well happen that after 0.2 we get rid of `IsHPC` completely. Then there's no need to rename that variable in all our `*.g` and `*.tst` files.

**Comment by ssiccha** [comment]
Now that we have changed the implementation we can't use a sequential version of this algorithm in HPC-GAP anymore. At least not without jumping through some hoops. So let's remove this parameter.
